cbbr_build/python/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sqlalchemy
```

Comment:
we don't actually need this, because they are included in the docker image
![image](https://user-images.githubusercontent.com/13207770/71183547-2f9b8d80-2246-11ea-91e4-ace0af26bb8f.png)